schema/mitwelten_v2.sql

```diff
@@ -201,6 +201,40 @@ CREATE TABLE IF NOT EXISTS prod.files_entry
     UNIQUE (object_name)
 );
 
+CREATE TABLE IF NOT EXISTS prod.image_results (
+  result_id serial PRIMARY KEY,
+  file_id int,
+  config_id varchar
```

Comment:
The config ID has its source here [model_config.json](https://github.com/mitwelten/pollinator-ml-backend/blob/main/model_config.json). The idea was to adjust it locally instead of requesting the ID at each iteration of a flow and was meant to be like a more descriptive ID... i.e. confthresh0.5_iou0.5_margin15.

Reply:
OK.

Reply:
Concerning the constraints on foreign-key fields. I will add it for all important fields. Thanks for the hint.